apps/web/src/components/notebook/index.js

```diff
@@ -154,6 +155,18 @@ const menuItems = [
     iconColor: "error",
     icon: Icon.Trash,
     onClick: async ({ items }) => {
+      if (await showDeleteNotesDialog("Notebook")) {
+        items.map(async (item) => {
+          db.notebooks.notebook(item.id).topics.all.map(async (topic) => {
+            await db.notebooks
+              .notebook(item.id)
+              .topics.topic(topic.id)
+              .all.map(async (note) => {
```

Comment:
You can assign `db.notebooks.notebook(item.id).topics` to a variable and then reuse it.